quickstart/aws/tutorial-eks.md

```diff
@@ -0,0 +1,303 @@
+	+   │  ├─ pulumi-nodejs:dynamic:Resource          helloworld-cfnStackName             	create
+	+   │  ├─ aws:ec2:SecurityGroup                   helloworld-eksClusterSecurityGroup  	create
+	+   │  ├─ aws:iam:InstanceProfile                 helloworld-instanceProfile          	create
+	+   │  ├─ aws:eks:Cluster                         helloworld-eksCluster               	create
+	+   │  ├─ pulumi-nodejs:dynamic:Resource          helloworld-vpc-cni                  	create
+	+   │  ├─ pulumi:providers:kubernetes             helloworld-eks-k8s                  	create
+	+   │  ├─ aws:ec2:SecurityGroup                   helloworld-nodeSecurityGroup        	create
+	+   │  ├─ kubernetes:core:ConfigMap               helloworld-nodeAccess               	create
+	+   │  ├─ kubernetes:storage.k8s.io:StorageClass  helloworld-gp2                      	create
+	+   │  ├─ aws:ec2:SecurityGroupRule               helloworld-eksClusterIngressRule    	create
+	+   │  ├─ aws:ec2:LaunchConfiguration             helloworld-nodeLaunchConfiguration  	create
+	+   │  ├─ aws:cloudformation:Stack                helloworld-nodes                    	create
+	+   │  └─ pulumi:providers:kubernetes             helloworld-provider                 	create
+	+   └─ aws-infra:network:Network                  vpc                               	create
+	+      ├─ aws:ec2:Vpc                             vpc                               	create
+	+      ├─ aws:ec2:Eip                             vpc-nat-0                         	create
+	+      ├─ aws:ec2:Eip                             vpc-nat-1                         	create
+	+      ├─ aws:ec2:InternetGateway                 vpc                               	create
+	+      ├─ aws:ec2:Subnet                          vpc-nat-1                         	create
+	+      ├─ aws:ec2:Subnet                          vpc-0                             	create
+	+      ├─ aws:ec2:Subnet                          vpc-nat-0                         	create
+	+      ├─ aws:ec2:Subnet                          vpc-1                             	create
+	+      ├─ aws:ec2:RouteTable                      vpc                               	create
+	+      ├─ aws:ec2:NatGateway                      vpc-nat-1                         	create
+	+      ├─ aws:ec2:RouteTableAssociation           vpc-nat-1                         	create
+	+      ├─ aws:ec2:NatGateway                      vpc-nat-0                         	create
+	+      ├─ aws:ec2:RouteTableAssociation           vpc-nat-0                         	create
+	+      ├─ aws:ec2:RouteTable                      vpc-nat-1                         	create
+	+      ├─ aws:ec2:RouteTable                      vpc-nat-0                         	create
+	+      ├─ aws:ec2:RouteTableAssociation           vpc-1                             	create
+	+      └─ aws:ec2:RouteTableAssociation           vpc-0                             	create
+
+	Resources:
+		+ 42 to create
+
+	clusterng (eks-demo):
```

Comment:
Maybe make a note here that you should select "yes" to confirm the deployment. I'd probably just copy whatever we do in other tutorials.